backend/prisma/migrations/20240705002820_updated_boards/migration.sql

```diff
@@ -0,0 +1,160 @@
+/*
```

Comment:
Are these warnings pre-generated or did you put them in here intentionally?

Reply:
pre-generated. Not sure how to resolve them. Will look into that

Reply:
If they're generated, I think it's fine to keep them there!